docs/ruleset_functions/get_building_segment_SWH_bat.md

```diff
@@ -0,0 +1,64 @@
+## get_building_segment_SWH_bat
+
+Description: This function determines the SWH BAT for the given building segment.
+
+Inputs:
+- **RMD**
+- **building_segment**
+
+Returns:
+- **building_segment_swh_bat**: one of the ServiceWaterHeatingSpaceOptions2019ASHRAE901 options
+
+Function Call:
+
+- get_energy_required_to_heat_swh_use  
+- get_SWH_uses_associated_with_each_building_segment    
+- get_component_by_ID  
+
+Data Lookup: None
+
+Logic:
+
+- set the building_segment_swh_bat to nil: `building_segment_swh_bat = "UNDETERMINED"`
+- if the building_segment.service_water_heating_building_area_type exists set the swh_bat to the one given: `if building_segment.service_water_heating_building_area_type:`
+    - set the building_segment_swh_bat to the building_segment.service_water_heating_building_area_type: `building_segment_swh_bat = building_segment.service_water_heating_building_area_type`
+- otherwise, if the building segment doesnt have service_water_heating_building_area_type, we need to determine the building segment SWH type by looking at individual SWH uses: `else:`
```

Comment:
It will be very helpful to avoid using terms like "use type", and "SWH type". They are confusing terms because there is `ServiceWaterHeatingUse.water_serves_type` and `ServiceWaterHeatingUse.area_type`. Please be consistent in referring to it as "area type"